components/kvengine/src/table/sstable/builder.rs

```diff
@@ -16,11 +16,13 @@ pub const PROP_KEY_MAX_TS: &str = "max_ts";
 pub const PROP_KEY_ENTRIES: &str = "entries";
 pub const PROP_KEY_OLD_ENTRIES: &str = "old_entries";
 pub const PROP_KEY_TOMBS: &str = "tombs";
-pub const PROP_KEY_FUSE8: &str = "fuse8";
+pub const AUX_INDEX_BINARY_FUSE8: u8 = 1;
+pub const INDEX_FORMAT_V1: u8 = 1;
+pub const BLOCK_FORMAT_V1: u8 = 1;
 pub const NO_COMPRESSION: u8 = 0;
 pub const LZ4_COMPRESSION: u8 = 1;
 pub const ZSTD_COMPRESSION: u8 = 2;
-const TABLE_FORMAT: u16 = 1;
+pub const TABLE_FORMAT_V1: u16 = 1;
```

Comment:
Why not use table format only? as long as there is a format change, we increases it only.

Reply:
table format is on higher level, it's hard to describe the combination of each component's format.
So the table format will be used to define the layout of each components, each component can have there own format.